anemometer/smart-contract/contracts/AnemometerOracle.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.5.8;
+pragma experimental ABIEncoderV2;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "iexec-doracle-base/contracts/IexecDoracle.sol";
+
+contract AnemometerOracle is Ownable, IexecDoracle
+{
+	struct TimedValue
+	{
+		bytes32 oracleCallID;
+		uint256 date;
+		string value;
```

Comment:
values could be split info :
- string weather
- uint256 windspeed
a string is not really usefull to smart contracts

Reply:
Changed into multiple variables and accordingly the events and data structures. The contract is deployed at 0x86bd850D4122Cf59A395ae348b36F3ae59116A4A on Kovan testnet.